src/NzbDrone.Core/Datastore/Migration/018_remove_duplicates.cs

```diff
@@ -35,7 +35,7 @@ private void RemoveDuplicates(IDbConnection conn, IDbTransaction tran)
         {
             var getDuplicates = conn.CreateCommand();
             getDuplicates.Transaction = tran;
-            getDuplicates.CommandText = string.Format("select id, {0} from {1}", columnName, tableName);
+            getDuplicates.CommandText = string.Format("SELECT \"Id\", \"{0}\" from \"{1}\"", columnName, tableName);
```

Comment:
Yea, this all depends on how the string is concatenated, it's done differently in different places. I'll see about consolidating.

Reply:
Essentially this comes down to if strings are created with `string.format()`, `""` or `$""` then `\"` is used. If the string is concatenated with `@""` then `""` is required for escape.

Reply:
Yeah, at this point we should move away from `string.Format` in general, which means we'd most often use `$`, but right now we sometimes use that or sometimes `@` for these SQL strings. We could use `$@` (or is it `@$`?) if we want to use `""` instead of `\"`, but really just looking for it to be consistent so we can keep it consistent in the future.

Reply:
We'd need to move normal string literals as well if we want to use `""`, a string defined with just `"string"` uses `\"` for escape. Seems like verbatim strings (`@""`) and `""` escape are the outliers really. Are there cases where we have to use verbatim for SQL strings?

Reply:
I don't think I saw any when I was reviewing.